packages/wallets/src/components/Base/WalletPasswordField/PasswordViewerIcon.tsx

```diff
@@ -10,8 +10,9 @@ interface PasswordViewerIconProps {
 
 const PasswordViewerIcon: React.FC<PasswordViewerIconProps> = ({ setViewPassword, viewPassword }) => (
     <IconButton
+        className='wallets-password__icon'
         color='transparent'
-        icon={viewPassword ? <PasswordShow /> : <PasswordHide />}
+        icon={viewPassword ? <PasswordHide /> : <PasswordShow />}
```

Comment:
@aizad-deriv in my opinion wrong logic or prop name. If I see `viewPassword` equals to `true` then I suppose to see password, but you render `PasswordHide` instead.

Reply:
yep the naming is confusing, it is actually in reverse